composer.json

```diff
@@ -6,11 +6,12 @@
 
     , "require": {
           "php": "^5.5|^7.0"
-        , "illuminate/database": "~5.2.0"
-        , "symfony/dependency-injection": "^2.6|^3.0"
+        , "illuminate/database": "^5.0.0 <5.3"
+        , "symfony/dependency-injection": "^2.8|^3.0"
     }
     , "require-dev": {
           "matthiasnoback/symfony-dependency-injection-test": "^0.7"
+        , "phpunit/phpunit": "^4.8|^5.0"
```

Comment:
as PHPUnit executes your code, its deps may conflict if you code ones, as so it's better to guarantee they don't by including them in your package dep resolution